package.json

```
@@ -9,49 +9,51 @@
     "typography-theme-wordpress-2016": "^0.16.19"
   },
   "devDependencies": {
-    "@cspell/dict-pt-br": "^2.3.3",
-    "@testing-library/jest-dom": "^6.6.3",
-    "@testing-library/react": "^12.1.5",
     "@babel/core": "^7.22.0",
+    "@cspell/dict-pt-br": "^2.3.3",
+    "@testing-library/jest-dom": "^6.0.0",
+    "@testing-library/react": "^14.0.0",
     "babel-jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.0.0",
```

Biome (1.9.4)

[error] The key jest-environment-jsdom was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)